apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -241,6 +242,20 @@ class ManageStudentsTable extends Component {
     );
   };
 
+  projectSharingFormatter = (projectSharing, {rowData}) => {
+    let disabled = rowData.isEditing ?
```

Comment:
When this gets toggled with an icon, it might get complex enough that you'd want it in a separate component, but this looks good for this PR.